graphql/schema.graphql

```diff
@@ -1,16 +1,9 @@
 # To really understand GraphQL, go to http://graphql.org/learn/
 
-# Autopersisted types are stored automatically in VTEX masterdata
-type AutopersistedBook @autopersist {
-  id: ID!                   # Autopersisted types must have an ID
-  name: String @searchable  # @searchable allows you to fetch a book by it's name
-  authors: [String!]        # This is an array of non-null strings
-}
-
 # Cache control can be set at each field
 # ref: https://www.apollographql.com/docs/engine/caching.html
 type CachedBook {
-  id: ID!
+  id: String
```

Comment:
ID is a first class citizen in GraphQL. It works exactly like a string, but it's a good abstraction for ids. Actually you can create other abstractions for your problem with GraphQL Scalars ! [check this out](https://www.apollographql.com/docs/graphql-tools/scalars.html)